SSA/Projects/CIRCT/DC/DC.lean

```diff
@@ -0,0 +1,184 @@
+import SSA.Core.Framework
+import SSA.Core.MLIRSyntax.EDSL
+import SSA.Projects.CIRCT.Stream.Stream
+
+namespace CIRCTStream
+namespace DC
+
+abbrev ValueStream := Stream
+abbrev TokenStream := Stream Unit
+
+def unpack (x : ValueStream α) : ValueStream α × TokenStream :=
```

Comment:
Yes I agree with this separation and I think the semantics we have so far is a good starting point for that, since `TokenStream` is explicitly different than `ValueStream` we can just use the two for different operations (control vs. data)